mmv1/third_party/terraform/services/sql/resource_sql_database_instance_test.go

```diff
@@ -2076,6 +2076,66 @@ func TestAccSqlDatabaseInstance_rootPasswordShouldBeUpdatable(t *testing.T) {
 	})
 }
 
+func TestAccSqlDatabaseInstance_SqlServerTimezone(t *testing.T) {
```

Comment:
We can remove `TestAccSqlDatabaseInstance_SqlServerTimezone` given it's covered in the update test `TestAccSqlDatabaseInstance_SqlServerTimezoneUpdate`